tests/rules/no-duplicate-headings.test.js

```diff
@@ -378,3 +378,12 @@ Heading 1
 		},
 	],
 });
+
+it("should not timeout for large inputs", () => {
```

Comment:
The test output now shows three tests with the same title. This is from https://github.com/eslint/markdown/actions/runs/16327872488/job/46158586794:

```text

  ✔ should not timeout for large inputs (347ms)
  ✔ should not timeout for large inputs (592ms)
  ✔ should not timeout for large inputs (581ms)
  examples
    react
      ✔ reports errors on code blocks in .md files (277ms)

  examples
    typescript
      ✔ reports errors on code blocks in .md files (685ms)

  MarkdownLanguage
    validateLanguageOptions()
```

To keep the tests distinct we could include the rule name in the title. It could be also helpful to add a comment with a link to the pull request before the test for reference.

```suggestion
// https://github.com/eslint/markdown/pull/463
it("`no-duplicate-headings` should not timeout for large inputs", () => {
```

Reply:
Makes sense, updated